src/index.js

```diff
@@ -1,12 +1,17 @@
-const { ApolloLink, Observable } = require('apollo-link')
 const {
+  ApolloLink,
+  Observable,
   selectURI,
   selectHttpOptionsAndBody,
   fallbackHttpConfig,
   serializeFetchParameter,
   createSignalIfSupported,
-  parseAndCheckHttpResponse
-} = require('apollo-link-http-common')
+  parseAndCheckHttpResponse,
+  fromError
+} = require('@apollo/client')
+const {
+  rewriteURIForGET
+} = require('@apollo/client/link/http/rewriteURIForGET')
```

Comment:
Unfortunately this deep import is not an option, as it contains ESM and not CJS:

https://unpkg.com/browse/@apollo/client@3.0.0-beta.31/link/http/rewriteURIForGET.js

Are you sure it is not exported in the package main CJS file?

https://unpkg.com/browse/@apollo/client@3.0.0-beta.31/apollo-client.cjs.js

If not, this needs to be fixed on Apollo's side.

Reply:
It does not seem to be. :(

Reply:
I wonder if we can try to bundle it as we build using rollup.js. Or we can ask it to be exported on Apollo's side then.

Reply:
Definitely needs to be fixed on Apollo's side. Due to workload I won't be able to champion this, would you like to? Maybe raise a PR over there.

Reply:
Can't the `rewriteURIForGET` stuff be done in a separate PR?

Reply:
I've removed it, as suggested it can be done in a separate PR once possible from Apollo's end.